src/main/java/executor/command/CommandBlank.java

```diff
@@ -7,8 +7,14 @@
 public class CommandBlank extends Command {
 
     // Constructor
+    /**
+     * Constructor for CommandBlank subCommand Class.
+     * @param userInput The user input from the CLI
+     */
     public CommandBlank(String userInput) {
         this.userInput = userInput;
+        this.description = "You can print a line separator";
```

Comment:
Since most of the other descriptions start with verbs (e.g. deletes, shows) I think it would be best if all descriptions begin with a verb. So change it to "Prints a line separator".